### ISSUE TITLE
Having issues lately

### ISSUE THREAD

**Issue by realjax** [dceejay/RedMap#80]
I'm having problems showing the map lately ( just a grey backround is shown).
I've upgraded node-red, the dashboard and this software to the lastest version. Running on node v8.15.

This is the console output of Chrome ( on MacOs) for the /ui:

> Refused to apply style from 'http://192.168.1.214:1880/worldmap/leaflet/Leaflet.vector-markers.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
> 
> 192.168.1.214/:39 GET http://192.168.1.214:1880/worldmap/leaflet/Leaflet.vector-markers.min.js net::ERR_ABORTED 404 (Not Found)
> 
> 192.168.1.214/:59 GET http://192.168.1.214:1880/worldmap/leaflet/tile.stamen.js net::ERR_ABORTED 404 (Not Found)
> 
> milsymbol.js:32 milsymbol.js 1.3.3 - Copyright (c) 2017 Måns Beckman  http://www.spatialillusions.com
> 192.168.1.214/:59 GET http://192.168.1.214:1880/worldmap/leaflet/tile.stamen.js net::ERR_ABORTED 404 (Not Found)
> 
> (index):193 CONNECT TO /worldmap/socket
> (index):235 IN an iframe
> dialog-polyfill.js:333 Can't upgrade <dialog>: already supported <dialog id=​"helpWindow">​…​</dialog>​
> dialogPolyfill.registerDialog @ dialog-polyfill.js:333
> (anonymous) @ (index):286
> (index):728 Uncaught TypeError: L.StamenTileLayer is not a constructor
>     at (index):728
> (anonymous) @ (index):728
> (index):157 CONNECTED

When I request the JS files (that generate a 404 ) directly I get them just fine.

**Comment by dceejay** [comment]
Most odd. Have you ensured you flushed the cache in the browser ?

**Comment by realjax** [comment]
Thanks, I did not mention it but yes, I did clear the cache. Or at least, I thought I did.  I use a chrome extension to clear my cache ( Im a web developer so clearing a cache needs to be as easy as pressing a button :))  and all though this extension has so far worked just fine, it didnt in this case and I had to resort to chrome's clear history settings, And then it worked.

As a side note.. On my Android phone I am also running Chrome and the map also didn't display any more in that browser. After I cleared the cache there It did start to work again, but after a few hours it stopped again, showing only a grey background again.  Strange..
